internal/cache/controller_obj_cache.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/third_party/github.com/vmware/alb-sdk/go/clients"
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/third_party/github.com/vmware/alb-sdk/go/session"
 
+	pq "github.com/jupp0r/go-priority-queue"
```

Comment:
Should we go with a pq implementation with more forks and stars such as https://github.com/oleiade/lane which has generics support?